### ISSUE TITLE
GPS sensor works, but "Invalid" in GUI

### ISSUE THREAD

**Issue by JaBecker** [achim1/muonic#79]
Steps to reproduce:
1. Install Muonic 2.1.2 on Ubuntu 14.04 on the provided laptop
2. Attach GPS sensor to DAQ box and wait for signal (green LED shows enough satellites)
3. open Muonic and go to "GPS output" and press "Show GPS"

Result: STATUS: Invalid, Checksum: Error, Satellites: 0

But if the command "DG" is entered as an DAQ command, correct data is displayed including the number of satellites. Also, the ChkSumErr counter is 0.

**Comment by hpbretz** [comment]
Did you install muonic through pip using the newest version? The development of muonic moved to a different repository, you can find it at https://github.com/CosmicLabDESY/muonic.

**Comment by hpbretz** [comment]
It would be great if you could check out the trunk from the develop branch in that repository and see if you get the same error.

**Comment by JaBecker** [comment]
Yes I installed it via the newest pip. I will test the version from the new repository after the cosmic-day.

**Comment by hpbretz** [comment]
I just checked, the version on pip is also very recent, so if confirmed,
this bug will be also in the most recent commit. Is it a problem for the
ICD?

2016-10-27 19:36 GMT+02:00 JaBecker notifications@github.com:

**Comment by JaBecker** [comment]
No, I dont think so. The GPS module is optional for the ICD.

Am 27. Oktober 2016 19:50:10 MESZ, schrieb hpbretz notifications@github.com:
